mpas_analysis/shared/interpolation/interpolate.py

```diff
@@ -82,19 +83,28 @@ def build_remap_weights(sourceFileName, outWeightFileName,
         does nothing and returns immediately, potentially saving a costly
         re-computaiton of the mapping file.
 
+    Raises
+    ------
+    OSError
+        If `ESMF_RegridWeightGen` is not in the system path.
+
     Author
     ------
     Xylar Asay-Davis
 
     Last Modified
     -------------
-    02/25/2017
+    03/14/2017
     """
 
     if not overwrite and os.path.exists(outWeightFileName):
         # a valid weight file already exists, so nothing to do
         return
 
+    if find_executable('ESMF_RegridWeightGen') is None:
+        raise OSError('ESMF_RegridWeightGen not found. Make sure esmf package '
+                      'is installed via\nlatest nco: \n    conda install nco')
```

Comment:
@milenaveneziani, I don't really think the command is wrong.  We could change it but conda-forge works differently than other channels in that it is meant to replace the main conda channel. Thus, the correct way to use it is to add is via:
```
conda config --add channels conda-forge
```
so that all packages, not just nco, are installed from conda-forge by preference.  See:
https://conda-forge.github.io/

We could add the above command to the error message but I think it belongs in the README.md (and will soon be there from my pending PRs), not in this error message.